src/index.html

```diff
@@ -1,13 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__photo">
+        <img
+          class="product-card__img"
+          src="../src/images/imac.jpeg"
+          alt="imac.jpeg"
+        >
+      </div>
+
+      <div class="product-card__article">
+        <a
+          class="product-card__link"
+          data-qa="hover"
+          href="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+            APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)
+        </a>
+      </div>
+
+      <div class="product-card__code">
+        Product code: 195434
+      </div>
+
+      <div class="stars">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <a class="stars__review" href="Review: 5">Reviews: 5</a>
```

Comment:
is this really should be a stars element?
```suggestion
        <p class="stars__review" href="Review: 5">Reviews: 5</a>
```